package.json

```
@@ -29,4 +30,4 @@
   "scripts": {
     "test": "NODE_ENV=abc jest"
   }
-}
+}
```

Trailing newline is preferred

```suggestion
}

```